examples/sugarscape_g1mt/sugarscape_g1mt/agents.py

```diff
@@ -266,8 +266,8 @@ def trade(self, other):
 
         # Our extra sanity check. The agent must still have enough resource
         # before the trade.
-        assert self.sugar > 0
-        assert self.spice > 0
+        # assert self.sugar > 0
```

Comment:
This constraints are important. Otherwise they can become negative.

Reply:
I am still working on this one it only seems to be throwing an error with the server but very irregularly.